test/parallel/test-inspector-invalid-args.js

```diff
@@ -1,24 +1,28 @@
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 
 common.skipIfInspectorDisabled();
 
 const assert = require('assert');
 const execFile = require('child_process').execFile;
 const path = require('path');
 
-const mainScript = path.join(common.fixturesDir, 'loop.js');
+const mainScript = path.join(fixtures.fixturesDir, 'loop.js');
```

Comment:
What do you think about using `fixtures.path('loop.js')` to replace `path.join(..., 'loop.js')` and make this even simpler?

Reply:
done. thank you!